src/reaper_osara.cpp

```diff
@@ -3756,6 +3778,53 @@ void cmdMoveToPrevItem(Command* command) {
 	}
 }
 
+void reportNudgeAndZoomStep(double nudgeTime) {
+	TimeFormat tf = TF_MILSEC;
+	if (nudgeTime >= 1.0) tf = TF_ROUNDSEC;
+	outputMessage(formatTime(nudgeTime, tf));
+}
+
+void nudgeAndZoomStep(int direction) {
```

Comment:
I don't think there's ever a chance that we will want to pass direction=0 here. If that's correct, I would change this to bool previous instead of int direction. That way, there's never any confusion that 0 is a valid value for direction.